### ISSUE TITLE
in command prompt  C/C++ Edit configure (UI)

### ISSUE THREAD

**Issue by djmanusudha** [microsoft/vscode-cpptools#8785]
Issue Type: <b>Performance Issue</b>

c/c++ edit ui is getting error(command c/c++Edit (UI) is not found)
it showing like this i download the extension c/c++(intellisense,ebugging etc.)
please solve this issue

Extension version: 1.8.2
VS Code version: Code 1.64.0 (5554b12acf27056905806867f251c859323ff7e9, 2022-02-03T04:22:20.678Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-8130U CPU @ 2.20GHz (4 x 2208)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.84GB (2.74GB free)|
|Process Argv|--crash-reporter-id b2b8e4f2-3d72-431f-85c9-1a61b8e60750|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	  2218	   360	code main
    0	   114	  2592	   extensionHost
    0	    49	  8432	   crashpad-handler
    0	    95	  9360	   shared-process
    0	    69	  6544	     ptyHost
    0	    80	 11108	     fileWatcher
    0	   138	 11664	   gpu-process
    0	    51	 12376	   utility-network-service
    1	    85	 12428	   issue-reporter
    0	   169	 13100	   window (new.cpp - c++ prgm - Visual Studio Code)
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (new.cpp - c++ prgm - Visual Studio Code)
|    Folder (c++ prgm): 1 files
|      File types: cpp(1)
|      Conf files:;
```

</details>
<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30425749
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dsc:30433058
pythonvs932:30410667
wslgetstartedc:30433508
vs360:30404995
vsrem710cf:30416617
vscop841cf:30430978
vscscmwlcmc:30433760

```

</details>

**Comment by sean-mcmanus** [comment]
This may be a regression from PR https://github.com/microsoft/vscode-cpptools/pull/8505 .

@Colengms Can you add back the tempCommand stuff? It's supposed to queue the command to a temp list and play it back after the command has been fully registered later on.

**Comment by sean-mcmanus** [comment]
@djmanusudha Using the Edit Configuration (UI) command a 2nd time fixes the issue, right?

**Comment by michelleangela** [comment]
The fix for this issue is now available in version Insiders 1.9.0.
